service/templates/authorize.html

```diff
@@ -3,33 +3,37 @@
 {% block content %}
 <main class="s-form-page">
   <form action="" method="post" class="s-form s-form--login">
-    <h1> Tapis Authorization - Tenant {{ tenant_id }}</h1>
-    <p> You are logged in as user {{ username }}. The client application {{ client_display_name }} is requesting a Tapis
-        access token on your behalf.</p>
+    <h1>Tapis Authorization - Tenant {{ tenant_id }}</h1>
+    <p>You are logged in as user <var>{{ username }}</var>. The client application <var>{{ client_display_name }}</var> requests a Tapis access token on your behalf.</p>
```

Comment:
**After.** Wrapped variables in a tag, so they can be styled. Browser default style is italic, which is okay. I'll let designer review.